### ISSUE TITLE
How to get the same result after each execution?

### ISSUE THREAD

**Issue by dsy412610** [fchollet/keras#3115]
Hello,
I imported numpy and used `np.random.seed(4)` to generate the same random numbers which can make it. But this method didn't work after I used cuDNN and I don't know why. Who can help me? Thanks!

**Comment by fchollet** [comment]
Because cuDNN introduces stochasticity of its own. You can force determinism, but your model will run a lot slower. For Theano, see: http://deeplearning.net/software/theano/library/sandbox/cuda/dnn.html

**Comment by dsy412610** [comment]
Thank you! I disabled cuDNN, but it's weird that my model overfit earlier than before.
